readthedocs/projects/tasks.py

```diff
@@ -984,3 +984,8 @@ def clear_html_artifacts(version):
     if isinstance(version, int):
         version = Version.objects.get(pk=version)
     remove_dir(version.project.rtd_build_path(version=version.slug))
+
+
+@task()
```

Comment:
Without specifying the queue on this task, it will execute on the build instances, which have no access to the database. Specify `@task(queue='web')` to ensure tasks are executed on the right instances.